react/.gitignore

```diff
@@ -0,0 +1,131 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Simplify this to not include what's in the gitignore_global, and only what's necessary for React in dev.

Reply:
Should be resolved in https://github.com/radiasoft/sirepo/pull/4667/commits/9cc4554f84321a4ebad33583e8e797f7fec20a7d